presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/rtree/TestUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial.rtree;
+
+import com.esri.core.geometry.Point;
+import com.esri.core.geometry.ogc.OGCGeometry;
+import com.facebook.presto.geospatial.Rectangle;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static com.esri.core.geometry.ogc.OGCGeometry.createFromEsriGeometry;
+
+public final class TestUtils
+{
+    private TestUtils() {}
+
+    public static OGCGeometry makePoint(double x, double y)
+    {
+        return createFromEsriGeometry(new Point(x, y), null);
+    }
+
+    public static double randomDouble(Random rng, double minVal, double maxVal)
```

Comment:
- Don't abbreviate
- rng -> random, minVal -> min, maxVal -> max
- makeRandomRectangle -> makeRectangle; make it private
- makeRandomRectangles -> makeRectanges

```
public static List<Rectangle> makeRectangles(Random random, int count)
    {
        return IntStream.range(0, count)
                .mapToObj(i -> makeRectangle(random))
                .collect(toImmutableList());
    }

    private static double randomDouble(Random random, double min, double max)
    {
        return min + random.nextDouble() * (max - min);
    }

    /*
     * Make a random rectangle "near" origin of size "about" size.
     */
    private static Rectangle makeRectangle(Random random)
    {
        double minX = randomDouble(random, -100, 100);
        double minY = randomDouble(random, -100, 100);
        double sizeX = randomDouble(random, 0.0, 10);
        double sizeY = randomDouble(random, 0.0, 10);
        return new Rectangle(minX, minY, minX + sizeX, minY + sizeY);
    }
```